src/scss/components/tabset/_index.scss

```diff
@@ -137,13 +137,13 @@ hx-tabset:not([hx-defined]) {
 hx-tabset.beta-hxBound {
   hx-tab[current] {
     background-color: $gray-0;
-    background-color: var(--hxTabset-hxTab-current-bgcolor, $gray-0);
+    background-color: var(--hxTabset-hxTab-current-bgcolor, $gray-0);//there is no token for bg color
   }
 
   hx-tabcontent {
     background-color: $gray-0;
-    background-color: var(--hxTabset-hxTabcontent-bgcolor, $gray-0);
+    background-color: var(--hxTabset-hxTabcontent-bgcolor, $gray-0);//there is no token for bg color
     border-color: $gray-300;
-    border-color: var(--hxTabset-hxTabcontent-border-color, $gray-300);
+    border-color: var(--hxTabset-hxTabcontent-border-color, $gray-300);// there is no token for border color
```

Comment:
Please update to use design tokens. If lower level definition missing, use `$rax-color-gray-300`.

Keep the comment and add a space after the semicolon.

Reply:
Changes have been done, please review.